kdd-2024/ibis-tutorial-KDD2024.md

```diff
@@ -0,0 +1,273 @@
+
+Notes: 
+- I'll collect the extra info separately given that this is a public repo. 
+- For the in person tutorial I was thinking in Gil and I, at the moment. Not sure 
+  what the budget will be, or the situation. We need at least 2 people to run a 3h 
+  tutorial.
+ 
+Naty Clementi
+Gil Forsyth 
+
+# Tutors’ short bio and expertise related to the tutorial (up to 200 words per tutor)
+
+1. List of in-person presenters, i.e., tutors who will attend KDD and present 
+   part of the tutorial
+
+- Naty Clementi
+- Gil Forsyth 
+  
+
+### Naty Clementi
+
+**Bio**
+Naty Clementi is a senior software engineer at Voltron Data, contributing to Ibis full time. She is a former academic with a Masters in Physics and PhD in Mechanical and Aerospace Engineering. She is also an active member of Pyladies and volunteers as one of the directors of Women Who Code in the Washington DC network. She is originally from Argentina, and speaks both Spanish and English fluently. 
+
+**Expertise:** 
+Naty Clementi is an experienced instructor and educator. She has presented tutorials at meetups and conferences such as SciPy, PyData NYC, PyData Seattle, PyCon US. She has also taught multiple Python courses to graduate and undergraduate engineering students at George Washington University. 
+
+The most recent Ibis tutorial is the accepted Ibis tutorial at PyCon US 2024: 
+[Tutorials: Introduction to Ibis: blazing fast analytics with DuckDB, Polars, Snowflake, and more, from the comfort of your Python repl.](https://us.pycon.org/2024/schedule/presentation/55/)
+
+
+### Gil Forsyth
+
+**Bio**
+Gil Forsyth is a staff software engineer at Voltron Data. He followed the common career path of Japanese language specialist -> administrative assistant -> mechanical engineer -> computational fluid dynamicist -> data scientist -> software engineer -> machine learning engineer -> software engineer. Gil contributes to several projects in the PyData ecosystem and is a core maintainer of xonsh and Ibis.
+
+**Expertise:** 
+Gil Forsyth is an experienced instructor, having led tutorials at several PyData conferences, PyCon, and SciPy. He also ran internal training on Python and distributed data analysis at Capital One for several years. Gil is one of the core maintainers of Ibis.
+
+The most recent Ibis tutorial is the accepted Ibis tutorial at PyCon US 2024: 
+[Tutorials: Introduction to Ibis: blazing fast analytics with DuckDB, Polars, Snowflake, and more, from the comfort of your Python repl.](https://us.pycon.org/2024/schedule/presentation/55/)
+
+
+2. List of contributors, i.e., tutors who will only help prepare the tutorial material
+
+- I'm not sure who to put here, I will need designated responsible people for some 
+  Ibis ML showcase and Ibis streaming (Chloe?)
+
+
+# Corresponding tutor with her/his email address
+
+- Naty Clementi (I'll add my email upon submission)
+
+# Tutorial outline. Please provide as much detail as possible.
```

Comment:
This is the part that needs more work and help to be cohesive and  include streaming and ML.